tests/solver_tester.py

```diff
@@ -6,46 +6,14 @@
 
 import torch
 
+from benchmarks.systems.system import ClosedSystem, OpenSystem
```

Comment:
Why are you importing from the benchmarks module? It should be independent from the tests. I think you mean to import from `sesolve.closed_system.py` and `mesolve.open_system.py`.